doc/api/console.markdown

```diff
@@ -1,12 +1,21 @@
-# console
+# Console
 
     Stability: 2 - Stable
 
+The module defines a `Console` class and exports a `console` object.
+
+The `console` object is a special instance of `Console` that the output is
+sent to stdout or stderr.
+
+For easily to use it, `console` is a global object, so you can use it directly.
```

Comment:
"For ease of use, `console` is defined as a global object and can be used directly without `require`."